lib/ansible/modules/windows/win_psexec.py

```diff
@@ -0,0 +1,142 @@
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'community',
+                    'version': '1.0'}
+
+DOCUMENTATION = """
+---
+module: win_psexec
+version_added: '2.3'
+short_description: Runs commands (remotely) as another (privileged) user
+description:
+- Run commands (remotely) through the PsExec service
+- Run commands as another (domain) user (with elevated privileges)
+options:
+  command:
+    description:
+    - The command line to run through PsExec (limited to 260 characters).
+    required: true
+    default: whoami.exe
+  executable:
+    description:
+    - The location of the PsExec utility (in case it is not located in your PATH).
+    default: psexec.exe
+  hostnames:
+    description:
+    - The hostnames to run the command.
+    - If not provided, the command is run locally.
+  username:
+    description:
+    - The (remote) user to run the command as.
+    - If not provided, the current user is used.
+  password:
+    description:
+    - The password for the (remote) user to run the command as.
+    - This is mandatory in order authenticate yourself.
+    required: true
+  chdir:
+    description:
+    - Run the command from this (remote) directory.
+  noprofile:
+    description:
+    - Run the command without loading the account's profile.
+  elevated:
+    description:
+    - Run the command with elevated privileges.
+  limited:
+    description:
+    - Run the command as limited user (strips the Administrators group and allows only privileges assigned to the Users group).
+  system:
+    description:
+    - Run the remote command in the System account.
+  priority:
+    description:
+    - Used to run the command at a different priority.
+    choices:
+    - background
+    - low
+    - belownormal
+    - abovenormal
+    - high
+    - realtime
+  timeout:
+    description:
+    - The connection timeout in seconds
+author: Dag Wieers (@dagwieers)
+"""
+
+EXAMPLES = r'''
+# Test the PsExec connection to the local system with your user (runs whoami)
+- win_psexec:
+    password: some_password
+
+# Run the setup.exe installer on multiple servers using the Domain Administrator
+- win_psexec:
+    command: E:\setup.exe /i /IACCEPTEULA
+    hostnames:
+    - remote_server1
+    - remote_server2
+    username: DOMAIN\Administrator
+    password: some_password
+    priority: high
+
+# Run PsExec from custom location C:\Program Files\sysinternals\
+- win_psexec:
+    command: netsh advfirewall set allprofiles state off
+    executable: C:\Program Files\sysinternals\psexec.exe
+    hostnames: [ remote_server ]
+    password: some_password
+    priority: low
+'''
+
+RETURN = '''
+cmd:
```

Comment:
The only reason for me to output this was to be sure win_psexec was doing the right thing. And I can imagine some people will need the same thing if they have issues using it (wrt. quoting etc.). So that you can copy&paste and do exactly what the module did.